--- extendj/java5/parser/java14fix.parser	2017-05-30 08:09:08.392274954 +0200
+++ patches/java14fix.parser	2017-05-30 08:08:30.649275724 +0200
@@ -66,7 +66,7 @@
   | shift_expression_nn.e1 URSHIFT additive_expression.e2 {: return new URShiftExpr(e1, e2); :}
   ;
 Expr relational_expression_nn =
-    shift_expression_nn
+    dl_quantifier_expression_nn
   | name.e1 LT shift_expression.e2                          {: return new LTExpr(e1, e2); :}
   | shift_expression_nn.e1 LT shift_expression.e2           {: return new LTExpr(e1, e2); :}
   | name.e1 GT shift_expression.e2                          {: return new GTExpr(e1, e2); :}
@@ -109,7 +109,7 @@
  * for instanceof_expression_nn.
  */
 Expr relational_expression :=
-    shift_expression @BOTTOM
+    dl_quantifier_expression @BOTTOM
   | relational_expression.e1 LT shift_expression.e2     {: return new LTExpr(e1, e2); :}
   | relational_expression.e1 GT shift_expression.e2     {: return new GTExpr(e1, e2); :}
   | relational_expression.e1 LTEQ shift_expression.e2   {: return new LEExpr(e1, e2); :}
